kolibri/content/constants/presets.py

```diff
@@ -0,0 +1,29 @@
+"""
+This module contains constants representing the ids of FormatPreset.
+"""
+from django.utils.translation import ugettext_lazy as _
+
+# constants for FormatPreset
+HIGH_RES_VIDEO = "high_res_video"
+MID_RES_VIDEO = "mid_res_video"
+LOW_RES_VIDEO = "low_res_video"
+VECTOR_VIDEO = "vertor_video"
+
+EN_SUBTITLE = "en_subtitle"
+ES_SUBTITLE = "es_subtitle"
+ZH_SUBTITLE = "zh_subtitle"
```

Comment:
DHH users do not **need** to know whether the captions they are reading are the original transcript or a translation, they just need those captions to be the correct ones for the video they are viewing.
   
Bear with me while I recap... The purpose of the caption is to aid/help or enable comprehension of the video content. For that to happen, the captions we present to the user must be:

a) original transcript of that video 
OR 
b) translation of the original transcript in another language which they understand

It seems straightforward enough, but if we do not unequivocally link each caption (weather it's the original transcript or its translation) to its originating video, we may end up in the situation we had in KA Lite all up to release 0.15, where we were serving subtitles in Spanish to users who were viewing videos in that language (**just because they were in the same language**), while those subtitles were, in fact, translations of English video transcripts and therefore almost completely unusable with videos in Spanish (valid for all languages that had so-called "dubbed" videos). 

You may argue that it is an edge case, but it's one we already had to deal with, so let's make sure we prevent it in the future. Again, I am not sure this is actually the right place to do it, and I'm only pressing the point to make sure that we do not use **"in the same language as video"** as the **ONLY** criteria to display captions.  

@66eli77 @jamalex